about.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+  </head>
+  <body>
+
+    <!-- ......socials....// -->
+    <div class="socials">
+      <ul class="socials1">
+        <li class="social-img icon">
+          <img src="images/facebook.png" alt="twitter-icon" class="twitter">
+        </li>
+        <li class="social-img icon">
+          <img src="images/twitter.png" alt="linkedin-icon">
+        </li>
+        <li class="social-text icon">
+          <p>English</p>
+        </li>
+        <li class="social-text icon">
+          <p>My Page</p>
+        </li>
+        <li class="social-text icon">
+          <p>Logout</p>
+        </li>
+      </ul>
+
+    </div>
+
+    <!-- ....//nav-bar..../// -->
+
+    <nav class="navbar navbar-expand-lg bg-light">
+  <div class="container-fluid">
+    <a href="./index.html" class="arta"><img src="images/red.png" alt="" class="art"></a>
+    <!-- <img src="images/mother.png" alt="" class="art"> -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <!-- <div class="collapse navbar-collapse" id="navbarSupportedContent"> -->
```

Comment:
[OPTIONAL] Please kindly delete unused code fot it makes your code base look untidy